src/main/java/org/apache/commons/net/util/Base64.java

```diff
@@ -114,10 +114,10 @@ public class Base64 {
     // some state be preserved between calls of encode() and decode().
 
     /**
-     * Tests a given byte array to see if it contains only valid characters within the Base64 alphabet.
+     * Tests a given byte array to see if it contains any valid character within the Base64 alphabet.
```

Comment:
Oh, I had to read the code to confirm it. To me the updated text looks correct, great catch!

Reply:
thanks 😊